firefox-ios/Client/Frontend/Home/PrivateHome/PrivateMessageCardCell.swift

```diff
@@ -34,31 +34,22 @@ class PrivateMessageCardCell: UIView, ThemeApplicable {
     }
 
     private lazy var headerLabel: UILabel = .build { label in
-        label.font = DefaultDynamicFontHelper.preferredFont(
-            withTextStyle: .headline,
-            size: UX.labelFontSize
```

Comment:
thanks for the update @darkCavalier11! Seems like we can also remove `labelFontSize` entirely, since its not being used anymore.